config/names.yml

```diff
@@ -0,0 +1,608 @@
+names:
```

Comment:
I would probably get rid of the top level `names:` declaration so when access I didn't have to say `NAME_CONSTANTS['name']` everytime. Not a necessary change just a thought.